### DISCUSSION TITLE
Any plans to support playwright?

### DISCUSSION THREAD

**Discussion by adamchemiawppl** [sorry-cypress/sorry-cypress#868] [General]
Hi,

Are there any plans to support playwright, or is it currents.dev-only feature?

BR,
Adam

**Comment by agoldis** [comment]
There are plans to support PW. We are recovering from Cypress blocking and polishing the integration. Stay tuned!

**Comment by adamchemiawppl** [comment]
Great, thanks. Actually this move from cypress is what made me start re-working all tests in playwright today. Would love to keep using sorry-cypress with it though - waiting eagerly!

**Comment by adamchemiawppl** [comment]
Hey, since it has been some time from my question - any (even vague) timeline on when playwright support might be available?

**Reply by agoldis** [reply]
We've been overwhelmed by the recent blocking by Cypress, so had to reshuffle the roadmap. We'll announce as soon as we are ready and gained the focus back.

**Reply by adamchemiawppl** [reply]
Thank you, waiting for it - even a little uncertain hint would be handy :-)

**Reply by adamchemiawppl** [reply]
Any news/rough timelines on the topic?

**Comment by adamchemiawppl** [comment]
Hey, it has been way over a year. Any updates on that topic, or can we just assume "Sorry Playwright" is dead?

That would be a shame. I'd love to somehow contribute to the project, and currents.dev is not really an option for me for various reasons (one of which being there is no package for one-man-bands using playwright only on some of my projects).

**Reply by agoldis** [reply]
We will be merging sorry-cypress with Currents. See https://currents.dev/posts/may-2024-update. The work is in progress.

**Reply by adamchemiawppl** [reply]
Hey, that's great news!

However (unless I missed something) there is no mention of any actual timeline - even a very rough one. And TBH for me that's quite an issue, as I don't know if I should wait or develop my own solutions to problems I face.

Best regards,
Adam

**Reply by agoldis** [reply]
Hey, Adam, let's connect - I want to explore your use case and see what we can do. andrew at currents dot dev

**Reply by adamchemiawppl** [reply]
Sure, kontakt@chemiawp.pl
